specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2018-10-31-preview/dedicatedhsm.json

```diff
@@ -0,0 +1,606 @@
+        ]
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.HardwareSecurityModules/dedicatedHSMs": {
+      "get": {
+        "tags": [
+          "DedicatedHsms"
+        ],
+        "operationId": "DedicatedHsm_ListBySubscription",
+        "description": "The List operation gets information about the dedicated HSMs associated with the subscription.",
+        "parameters": [{
+            "name": "$top",
+            "in": "query",
+            "type": "integer",
+            "format": "int32",
+            "description": "Maximum number of results to return."
+          }, {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }, {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Get information about all dedicated HSMs in the specified subscription.",
+            "schema": {
+              "$ref": "#/definitions/DedicatedHsmListResult"
+            }
+          },
+          "default": {
+            "description": "The error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "ApiEntityReference": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ARM resource id in the form of /subscriptions/{SubcriptionId}/resourceGroups/{ResourceGroupName}/..."
+        }
+      },
+      "description": "The API entity reference."
+    },
+	  "NetworkInterface": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ARM resource id in the form of /subscriptions/{SubcriptionId}/resourceGroups/{ResourceGroupName}/..."
+        },
+		    "privateIpAddress": {
+          "type": "string",
+          "description": "Private Ip address of the interface"
+        }
+      },
+      "description": "The network interface definition."
+    },
+    "NetworkProfile": {
+      "properties": {
+        "subnet": {
+            "$ref": "#/definitions/ApiEntityReference",
+            "description": "Specifies the identifier of the subnet."
+        },
+        "networkInterfaces": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/NetworkInterface"
+            },
+            "description": "Specifies the list of resource Ids for the network interfaces associated with the dedicated HSM."
+        }
+      }
+    },
+    "Sku": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "SKU of the dedicated HSM",
+          "enum": [
+            "SafeNet Network HSM 7"
```

Comment:
This enum value doesn't match the code. Please sync with Nitin Goel

Reply:
Please use this: SafeNet Luna Network HSM A790